creepy/protocol/handshake.py

```diff
@@ -32,27 +32,36 @@ class HandshakeProtocol:
 
     def __init__(self, authorized_keys_path=None):
         self.salt = secrets.token_bytes(self.SALT_SIZE)
+        bobs = self._add_keys(authorized_keys_path)
+        self._bobs = bobs
+
+    def _add_key(self, key, bobs):
+        key_hash = self.pubkey_digest(key, self.salt)
+        bobs[key_hash] = Bob(key)
+
+    def _add_keys(self, authorized_keys_path):
```

Comment:
```suggestion
    def _load_authorized_keys(self, path):
```

Reply:
I think it's better to keep description of `path` for function signature to stay descriptive:
```
def _load_authorized_keys(self, keys_file_path):
```

Otherwise there may be question: 
* Path to what does it require? Is it folder with keys files or a file?

If we assume that `authorized_keys` usually means only one thing (the file created at `~/.ssh/authorized_keys` by default) and developers know what this file is, then we might stick with your suggestion without any corrections.

Reply:
I would prefer to just add docstring saying that `path` points to an `authorized_keys` file. But I'm also ok with your variant.